### DISCUSSION TITLE
Xdrip android Dexcom g7 not working

### DISCUSSION THREAD

**Discussion by ostapkohaluk** [NightscoutFoundation/xDrip#2876] [General]
![Screenshot_2023-06-07-10-59-00-354_com eveningoutpost dexdrip](https://github.com/NightscoutFoundation/xDrip/assets/135813578/9b306fee-6455-4b86-b840-b17e1c99c301)

**Comment by ostapkohaluk** [comment]
![Screenshot_2023-06-07-11-01-07-196_com eveningoutpost dexdrip](https://github.com/NightscoutFoundation/xDrip/assets/135813578/b8a13e79-93fc-43fb-9c93-1654098ea184)

**Comment by jamorham** [comment]
Can you provide more information? Which xDrip version are you using? (Try nightly version) - What steps have you taken so far? It looks like it was working and now has stopped? Is that correct? What does system status tab say?
